Game crashes when double click to mod.

For some mods, double clicking will crash game. (I tested on LazyDFU and some other mods, only LazyDFU is crashed my game)
```
Reported exception thrown!
net.minecraft.class_148: mouseClicked event handler
	at net.minecraft.class_437.method_25412(class_437.java:495)
	at net.minecraft.class_312.method_1601(class_312.java:94)
	at net.minecraft.class_312.method_22686(class_312.java:165)
	at net.minecraft.class_1255.execute(class_1255.java:101)
	at net.minecraft.class_312.method_22684(class_312.java:165)
	at org.lwjgl.glfw.GLFWMouseButtonCallbackI.callback(GLFWMouseButtonCallbackI.java:36)
	at org.lwjgl.system.JNI.invokeV(Native Method)
	at org.lwjgl.glfw.GLFW.glfwPollEvents(GLFW.java:3101)
	at com.mojang.blaze3d.systems.RenderSystem.flipFrame(RenderSystem.java:165)
	at net.minecraft.class_1041.method_15998(class_1041.java:310)
	at net.minecraft.class_310.method_1523(class_310.java:1137)
	at net.minecraft.class_310.method_1514(class_310.java:728)
	at net.minecraft.client.main.Main.main(Main.java:217)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:234)
	at net.fabricmc.loader.launch.knot.Knot.launch(Knot.java:153)
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:28)
Caused by: java.lang.NullPointerException: Cannot invoke "xyz.deathsgun.modmanager.api.mod.DetailedMod.body()" because "this.detailedMod" is null
	at xyz.deathsgun.modmanager.gui.ModDetailScreen.method_25426(ModDetailScreen.java:69)
	at net.minecraft.class_437.method_25423(class_437.java:387)
	at net.minecraft.class_310.method_1507(class_310.java:984)
	at xyz.deathsgun.modmanager.gui.ModsOverviewScreen.updateSelectedEntry(ModsOverviewScreen.java:119)
	at xyz.deathsgun.modmanager.gui.widget.better.BetterListWidget.setSelected(BetterListWidget.java:63)
	at xyz.deathsgun.modmanager.gui.widget.better.BetterListWidget$BetterListEntry.method_25402(BetterListWidget.java:211)
	at xyz.deathsgun.modmanager.gui.widget.better.BetterListWidget.method_25402(BetterListWidget.java:139)
	at net.minecraft.class_4069.method_25402(class_4069.java:27)
	at net.minecraft.class_312.method_1611(class_312.java:94)
	at net.minecraft.class_437.method_25412(class_437.java:490)
	... 19 more```

--- a/src/main/java/xyz/deathsgun/modmanager/gui/ModDetailScreen.java
+++ b/src/main/java/xyz/deathsgun/modmanager/gui/ModDetailScreen.java
@@ -63,7 +63,8 @@ public class ModDetailScreen extends Screen {
             detailedMod = ModManager.getModProvider().getMod(summarizedMod.id());
         } catch (Exception e) {
             e.printStackTrace();
-            Objects.requireNonNull(this.client).setScreen(new ModManagerErrorScreen(this, e));
+            Objects.requireNonNull(this.client).setScreen(new ModManagerErrorScreen(previousScreen, e));
+            return;
         }
         int buttonX = this.width / 8;
         String text = detailedMod.body().replaceAll("\\u00A0", " ");

